src/main/java/androidx/view/View.kt

```diff
@@ -171,3 +171,36 @@ fun View.toBitmap(config: Bitmap.Config = Bitmap.Config.ARGB_8888): Bitmap {
     }
     return Bitmap.createBitmap(width, height, config).applyCanvas(::draw)
 }
+
+/** Show current view */
+fun View.show() {
+    visibility = View.VISIBLE
+}
+
+/** Hide current view */
+fun View.hide() {
+    visibility = View.INVISIBLE
+}
+
+/** Gone current view */
+fun View.gone() {
```

Comment:
`inline`

Reply:
I'm a bit ambivalent of the naming of this method. show and hide are both verbs but gone isn't. I wonder if we could find a better name for consistency.

Reply:
How about `remove()`? It doesn't actually get removed from the view hierarchy, but it gets treated like it.  But the most straightforward names would be 

`View.setVisible()`
`View.setInvisible()`
`View.setGone()`

Reply:
.Net has Visibility Collapsed ([Visibility Enumeration](https://msdn.microsoft.com/en-us/library/ms590101.aspx)). Maybe it won't be intuitive but what about `collapse` ?

Reply:
IMO, `Collapsed` is way better name than `Gone`. However, unless we can time travel and do Refactor->Rename on the `GONE`constant ten year ago, we shouldn't introduce second name for the same concept.

Reply:
Maybe `View.goAway()`/`View.getOut()`?

Reply:
I understand that perhaps the name is not the best, but it is semantically close to the already declared meaning - GONE, to which we are all accustomed. I 100% agree with the opinion of @pchmielowski

Reply:
Maybe name them simply like so?
```
View.visible()
View.invisible()
View.gone()
```

Reply:
How about
```kotlin
View.makeVisible()
View.makeInvisible()
View.makeGone()
```

Reply:
Here is another one: 
```
View.show()
View.hide()
View.begone() //Still a verb unlike View.gone()
```

Reply:
Since none of the suggestions really work, let's keep `gone()` and add this to the long list of APIs we regret :)